Dockerfile

```
@@ -0,0 +1,53 @@
+    <provides>
+        <version>${BUILDER_JDK_MAJOR_VERSION}</version>
+        <vendor>${BUILDER_JDK_VENDOR}</vendor>
+        <id>${BUILDER_JDK_VENDOR}_${BUILDER_JDK_MAJOR_VERSION}</id>
+    </provides>
+    <configuration>
+        <jdkHome>${BUILDER_JDK_HOME_PATH}</jdkHome>
+    </configuration>
+    </toolchain>
+</toolchains>
+EOF
+RUN if [ -z "${USE_PREBUILT_ZIP:-}" ]; then \
```

Hadolint (2.14.0)

[error] unexpected 'R'
expecting a new line followed by the next instruction

(DL1000)